server/passport-providers.js

```diff
@@ -159,5 +159,20 @@ module.exports = {
     scope: ['email'],
     link: true,
     failureFlash: true
+  },
+  'github-repo': {
```

Comment:
Why does this need a second github auth? We are trying to reduce not add more auths